content/docs/1.3.1/advanced-resources/data-recovery/recover-without-system.md

```diff
@@ -28,3 +41,23 @@ This command gives users the ability to restore a backup to a `raw` image or a `
         kubectl -n longhorn-system get pod restore-to-file -w
 
 After the pod status changed to `Completed`, you should able to find `<OUTPUT_FILE>` at e.g. `/tmp/restore` on the `<NODE_NAME>`.
+
+
+
+ ## Appendix
+
```

Comment:
Can you mention the purpose of the section?

Reply:
Since I wanted to list the pros and cons between qcow2 and raw , I thought this was an extra info for reference.

Therefore, I have added an "Appendix" section for comparison.

Reply:
I feel this is out of scope because we don't really want to do any comparison between types here even though it was mentioned in the ticket. 

The page here is for `Recovering from a Longhorn Backup without System Installed`, so beyond that, it's not quite important.